webpack.config.js

```diff
@@ -2,56 +2,74 @@ const path = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const Dotenv = require("dotenv-webpack");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
+const BundleAnalyzerPlugin =
+  require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
```

Comment:
BundleAnalyzerPlugin 를 남겨두면  `npm run serve` 시에 분석 웹사이트도 같이 뜨지 않나욤?

BundleAnalyzerPlugin 를 설치한 건 별도의 브랜치로 관리하는 건 어떨까요~?

```suggestion
const { BundleAnalyzerPlugin } =
  require("webpack-bundle-analyzer");
```

Reply:
별도의 브랜치로 관리해보는게 이번이 처음이라서 새롭네요 👀 다음에는 무작정 삭제하는게 아니라 브랜치별로 작업을 나누는 연습도 혼자하는 프로젝트에서도 적용하면 좋을 것 같네용!! 현재는 배포브랜치라서 삭제했습니다! 53b873c70621ff91fb9a1f667310ac7ca7051f13